Assets/MixedRealityToolkit/InputSystem/Sources/TouchscreenInputSource.cs

```diff
@@ -23,7 +23,7 @@ public class TouchscreenInputSource : BaseInputSource
     {
         private const float K_CONTACT_EPSILON = 2.0f / 60.0f;
 
-        public override InputType[] Capabilities => new[] { InputType.Pointer, InputType.PointerPosition };
+        public override Dictionary<InputType, InteractionDefinition> Interactions => new Dictionary<InputType, InteractionDefinition>() { { InputType.Pointer, new InteractionDefinition() { InputType = InputType.Pointer, AxisType = AxisType.DualAxis, Changed = false, Id = InputSystem.GenerateNewSourceId().ToString() } } };
```

Comment:
Why are we setting the interaction definitions id to a generated id for an input source from the manager?